website/siteConfig.js

```diff
@@ -56,7 +56,10 @@ module.exports = {
     twitter: true,
     twitterImage: 'img/webdriverio.png',
     twitterUsername: 'webdriverio',
-    scrollToTop: true
+    scrollToTop: true,
+
+    /* Show last update time */
+    enableUpdateTime: true
```

Comment:
@WillBrock @christian-bromann this one doesn't work for me :(

Reply:
@mgrybyk hmm, I was able to get the updated date to display by setting `enabeUpdateTime: true` and then re-generating the docs.

![wdio-updated-at](https://user-images.githubusercontent.com/1300981/58381394-a14e7a00-7f8a-11e9-8fc9-e37ecbb0c7f6.png)

Reply:
If it works for you then it should work on site as well. Thanks again